[2024-12-05T22:54:47.720+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-05T22:54:47.732+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: brand_category_mapping.get_next_batch manual__2024-12-05T22:54:44.071844+00:00 [queued]>
[2024-12-05T22:54:47.738+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: brand_category_mapping.get_next_batch manual__2024-12-05T22:54:44.071844+00:00 [queued]>
[2024-12-05T22:54:47.739+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-05T22:54:47.751+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_next_batch> on 2024-12-05 22:54:44.071844+00:00
[2024-12-05T22:54:47.757+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8917) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-05T22:54:47.758+0000] {standard_task_runner.py:72} INFO - Started process 8918 to run task
[2024-12-05T22:54:47.759+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'brand_category_mapping', 'get_next_batch', 'manual__2024-12-05T22:54:44.071844+00:00', '--job-id', '1943', '--raw', '--subdir', 'DAGS_FOLDER/llm_mapping.py', '--cfg-path', '/tmp/tmp846t22rp']
[2024-12-05T22:54:47.760+0000] {standard_task_runner.py:105} INFO - Job 1943: Subtask get_next_batch
[2024-12-05T22:54:47.795+0000] {task_command.py:467} INFO - Running <TaskInstance: brand_category_mapping.get_next_batch manual__2024-12-05T22:54:44.071844+00:00 [running]> on host c7780884450d
[2024-12-05T22:54:47.857+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='artzsh' AIRFLOW_CTX_DAG_ID='brand_category_mapping' AIRFLOW_CTX_TASK_ID='get_next_batch' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T22:54:44.071844+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T22:54:44.071844+00:00'
[2024-12-05T22:54:47.858+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-05T22:54:47.883+0000] {base.py:84} INFO - Retrieving connection 'database_cloud'
[2024-12-05T22:54:48.758+0000] {llm_mapping.py:60} INFO - Все бренды обработаны. Завершение работы DAG.
[2024-12-05T22:54:48.759+0000] {python.py:240} INFO - Done. Returned value was: {'batch': [], 'next_index': 5850}
[2024-12-05T22:54:48.779+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-05T22:54:48.780+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=brand_category_mapping, task_id=get_next_batch, run_id=manual__2024-12-05T22:54:44.071844+00:00, execution_date=20241205T225444, start_date=20241205T225447, end_date=20241205T225448
[2024-12-05T22:54:48.815+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-05T22:54:48.833+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T22:54:48.836+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
